tests/integration-tests/tests/networking/test_placement_group.py

```diff
@@ -0,0 +1,196 @@
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    if cluster.config.get("cluster default", "placement_group") == "DYNAMIC":
+        # For tratitional scheduler, the placement group name can be retrieved from main stack, for slurm, it can be
+        # retrieved from ComputeFleetHITSubstack
+        if scheduler == "slurm":
+            compute_fleet_substack = utils.get_substacks(
+                cluster.cfn_name, region=region, sub_stack_name="ComputeFleetHITSubstack"
+            )[0]
+            compute_fleet_substack_resources = utils.retrieve_cfn_resources(compute_fleet_substack, region)
+            placement_group = next(
+                v for k, v in compute_fleet_substack_resources.items() if k.startswith("PlacementGroup")
+            )
+        else:
+            placement_group = utils.retrieve_cfn_resources(cluster.cfn_name, region)["DynamicPlacementGroup"]
+
+        assert_placement_group(cluster, scheduler, region, placement_group, remote_command_executor, None)
+
+    cluster.delete()
+
+
+@pytest.mark.regions(["eu-central-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm"])
+@pytest.mark.oss(["alinux2"])
+@pytest.mark.usefixtures("os", "instance", "scheduler", "region")
+def test_placement_group_in_queue(region, scheduler, pcluster_config_reader, clusters_factory, placement_group_stack):
+    """Test the case when placement_group is in queue section. This test is only for slurm."""
+    existing_placement_group = placement_group_stack.cfn_resources["PlacementGroup"]
+    cluster_config = pcluster_config_reader(placement_group=existing_placement_group)
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    compute_fleet_substack = utils.get_substacks(
+        cluster.cfn_name, region=region, sub_stack_name="ComputeFleetHITSubstack"
+    )[0]
+    compute_fleet_substack_resources = utils.retrieve_cfn_resources(compute_fleet_substack, region)
+    dynamic_placement_group = next(
+        v for k, v in compute_fleet_substack_resources.items() if k.startswith("PlacementGroup")
+    )
+    assert_placement_group(cluster, scheduler, region, dynamic_placement_group, remote_command_executor, "dynamic")
+    assert_placement_group(cluster, scheduler, region, existing_placement_group, remote_command_executor, "existing")
+    cluster.delete()
+
+
+def assert_placement_group(cluster, scheduler, region, placement_group, remote_command_executor, queue_name):
+    logging.info("Checking placement group")
+    actual_placement_group = get_placement_group(scheduler, region, cluster, queue_name)
+    assert_that(actual_placement_group).is_equal_to(placement_group)
+    if scheduler != "slurm":
+        try:
+            if (
+                cluster.config.get("cluster default", "placement")
```

Comment:
Don't like this general practice of getting things from the test config.
We generated the test config, so we should already know what is in it.
If a parameter can be different, let's make it a variable when we generate the config and pass it in here when we call the function.
Otherwise let's just use a static value in both the config and our tests